assets/src/dashboard/app/views/myStories/content/storyGridItem/components.js

```diff
@@ -127,20 +127,25 @@ export const CardDetailsColumn = styled.div`
 // This is supported by all major browsers except IE 11.
 // If it fails, max-height will catch so that really long titles
 // don't leave their designated space, just no ellipsis.
-export const Title = styled(Headline).attrs({
-  as: 'a',
+export const StoryTitle = styled(Headline).attrs({
   size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
 })`
   overflow: hidden;
   text-overflow: ellipsis;
-  color: ${({ theme }) => theme.colors.inverted.fg.primary};
+  color: ${({ theme }) => theme.colors.inverted.fg.tertiary};
   margin: 0;
   max-height: calc(1.2em * 3);
   /* stylelint-disable-next-line */
   display: -webkit-box;
   -webkit-line-clamp: 3;
   /* stylelint-disable-next-line */
   -webkit-box-orient: vertical;
+`;
+
+export const Title = styled(StoryTitle).attrs({
+  as: 'a',
+})`
+  color: ${({ theme }) => theme.colors.inverted.fg.primary};
```

Comment:
Hmm it's not super obvious that one is a link and the other isn't.

Can we rename them to something like `Title` and `TitleLink` to make that more clear?